tools/explorer/lib/model.py

```diff
@@ -47,25 +77,93 @@ class EntityTypeField(StandardField):
   def __init__(self,
                namespace_name: str,
                standard_field_name: str,
-               is_required: bool,
+               is_optional: bool,
```

Comment:
docstring deleted? also semantically seems like this flag is a breaking change?

Reply:
@kswang2400 This is actually fix from what was a breaking change. Since the FieldParts tuples store optionality where True is optional. I adjusted my class to better fit that tuple. The goal is to be able to convert optwrapper and fieldparts tuples to objects.